Material.Styles/Card.xaml

```diff
@@ -8,22 +8,66 @@
         <Setter Property="assists:ShadowAssist.ShadowDepth" Value="Depth2" />
         <Setter Property="Focusable" Value="False" />
         <Setter Property="ClipToBounds" Value="False" />
+        <Setter Property="Padding" Value="8"/>
         <Setter Property="Template">
             <ControlTemplate>
                 <Grid Background="Transparent" ClipToBounds="False">
-                    <Border ClipToBounds="False"
-                        assists:ShadowAssist.ShadowDepth="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(assists:ShadowAssist.ShadowDepth)}"
-                        CornerRadius="{TemplateBinding CornerRadius}">
-                        <Border Background="{TemplateBinding Background}" Padding="{TemplateBinding Padding}"
-                                x:Name="PART_ClipBorder" />
+                    <Border assists:ShadowAssist.ShadowDepth="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(assists:ShadowAssist.ShadowDepth)}"
+                            CornerRadius="{TemplateBinding CornerRadius}"
+                            ClipToBounds="False">
+                        <Border assists:ShadowAssist.ShadowDepth="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(assists:ShadowAssist.ShadowDepth)}"
+                                Background="{TemplateBinding Background}" 
+                                Padding="{TemplateBinding Padding}"
+                                x:Name="PART_ClipBorder" 
+                                CornerRadius="{TemplateBinding CornerRadius}"
+                                ClipToBounds="True">
+                          <ContentPresenter
+                                  x:Name="ContentPresenter"
+                                  Margin="{TemplateBinding Padding}"
+                                  Content="{TemplateBinding Content}"
+                                  ContentTemplate="{TemplateBinding ContentTemplate}" />
+                        </Border>
                     </Border>
-                    <ContentPresenter
-                        x:Name="ContentPresenter"
-                        Margin="{TemplateBinding Padding}"
-                        Content="{TemplateBinding Content}"
-                        ContentTemplate="{TemplateBinding ContentTemplate}" />
                 </Grid>
             </ControlTemplate>
         </Setter>
     </Style>
+
+  <Style Selector="cc|Card.NoClip">
```

Comment:
Why is this style needed?

Reply:
My mistake, I forgot to remove this style. I'm tring integrate card to popup but failed. This style can be removed safety.